README.md

```diff
@@ -48,6 +49,9 @@ npx gocd-cypress --help
 
 # Configuration
 
+You should configure the behaviour trough package.json. See all options avaialable in the [CLI builder](https://github.com/adnovum/gocd-cypress/blob/dbcdec4aad24f9637214600d3be1b58025f4e8eb/src/cli-builder.ts#L32)
```

Comment:
I find this paragraph completely redundant in its current form:

* While the options can indeed be read from the TypeScript source, we also list them in the README below. If you wish to document the options this way, consider removing the duplication in the README.
* `package.json` is not the only file where you can configure the Cypress runner, as the paragraph below explains: 
    > This tool uses Cosmiconfig for the configuration options, see its
documentation about where you can define your options.

Reply:
- The README lists the command line options,  but not these configuration keys, e.g. `resultsFolder` or `reportsFolder` is not mentioned at all.
The README seems to describe options from https://github.com/adnovum/gocd-cypress/blob/3496defe9b8affa112f0a81820bd512dc3f5be36/src/config.ts 
but not directly from : https://github.com/adnovum/gocd-cypress/blob/dbcdec4aad24f9637214600d3be1b58025f4e8eb/src/cli-builder.ts#L32 

- good point about Cosmiconfig, its incorrect to specify package.json, i will clean that up

- It looks like all Cosmiconfig options work as a CLI options (the README also shows examples for this), and all "command-line options" work as Cosmiconfig too. (this is described by the README).
We can just add a list of available configuration properties, and mention that they work as Cosmiconfig or CLI, as preferred.

My goal is to make it easier to discover all possible configuration options. Right now you need to look around in the source for that.

Reply:
In general I'm fine with easier discovery, as long as the documentation remains coherent. Currently the README already asks the user to consult `npx gocd-cypress --help` for the command-line flags, and also tells that `All command-line options are also available as Cosmiconfig options.` Apart from that, we got 3 configuration options that do not work from the command line: https://github.com/adnovum/gocd-cypress/blob/3496defe9b8affa112f0a81820bd512dc3f5be36/src/config.ts#L7 . Those are explicitly mentioned in the README, although this is some kind of duplication currently that fails to maintain a single source of truth, and I only have a problem specifically with that. If we link the source we must make sure the argument listing in the source will remain in a declarative code style, otherwise it won't even have a chance to be suitable for documentation.